dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -90,10 +90,11 @@ public void testArtifacts()
       throws IOException, RepositoryException, TransformerException, XMLStreamException {
     try {
       LinkageCheckerMain.main(
-          new String[] {"-a", "com.google.cloud:google-cloud-firestore:0.65.0-beta"});
+          // protobuf-java:3.13.0 in
+          new String[] {"-a", "com.google.cloud:google-cloud-firestore:1.35.2"});
       fail("LinkageCheckerMain should throw LinkageCheckResultException upon errors");
     } catch (LinkageCheckResultException expected) {
-      assertEquals("Found 75 linkage errors", expected.getMessage());
+      assertEquals("Found 76 linkage errors", expected.getMessage());
```

Comment:
Diff:

Linkage Errors before the change (with 0.65.0-beta): https://gist.github.com/suztomo/b0a19b75b2a79960ab4a978b4ba70c8d
Linkage Errors after the change (with 1.35.2): https://gist.github.com/suztomo/9a44b2d6994200fd5fe3110830e0e01b